website/frontend/src/consts.ts

```diff
@@ -8,6 +8,21 @@ export const WASM_COMPILER_GET = `${process.env.REACT_APP_WASM_COMPILER_URL}/get
 export const isProd = process.env.NODE_ENV === 'production';
 export const INITIAL_LIMIT_BY_PAGE = 13;
 export const GEAR_BALANCE_TRANSFER_VALUE = 10_000_000_000;
+export const MIN_GAS_LIMIT = 20_000_000;
+
+export const META_FIELDS = [
+  'async_handle_input',
+  'async_handle_output',
+  'async_init_input',
+  'async_init_output',
+  'handle_input',
+  'handle_output',
+  'init_input',
+  'init_output',
+  'meta_state_input',
+  'meta_state_output',
+  'types',
+];
```

Comment:
I believe it's better to define it locally. Or to do it using enum so we'd have the ability to reuse names across other components.